rudder-core/src/main/resources/reportsSchema.sql

```diff
@@ -193,6 +197,66 @@ CREATE TABLE archivedNodeConfigurations (
 , PRIMARY KEY (nodeId, nodeConfigId, beginDate)
 );
 
+/* 
+ *************************************************************************************
+ * The following tables stores "node compliance", i.e all the interesting information
+ * about what was the compliance of a node FOR A GIVEN RUN. 
+ * That table *only* store information for runs, and does not track (non exaustively):
+ * - when the node expected configuration is updated - only a new run will check,
+ * - node not sending runs - only the fact that we don't have data can be observed
+ * - if a node is deleted
+ *************************************************************************************
+ */
+
+-- Create the table for the node compliance
+CREATE TABLE nodeCompliance (
+  nodeId            text NOT NULL CHECK (nodeId <> '')  
+, runTimestamp      timestamp with time zone NOT NULL
```

Comment:
i do agree with your remark that nodeconfigid should be in a dedicated column, I'm pretty sure it will be useful in many cases